client/modules/workspace/src/AppsBreadcrumb/AppsBreadcrumb.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import { Breadcrumb } from 'antd';
+import { Link, useParams, useLocation } from 'react-router-dom';
+import styles from './AppsBreadcrumb.module.css';
+import { TAppParamsType, TAppResponse, useGetApps } from '@client/hooks';
+
+function getPathRoutes(path: string = '') {
+  const pathComponents = decodeURIComponent(path)
+    .split('/')
+    .filter((p) => !!p);
+
+  return pathComponents.map((comp, i) => ({
+    title: comp,
+    path: '/' + encodeURIComponent(pathComponents.slice(0, i + 1).join('/')),
```

Comment:
nit: In the [design](https://xd.adobe.com/view/bab9a62d-3f2a-45f8-be0f-5680c474d9c1-efea/screen/ad7f3fda-3bc8-4aee-86ba-75d5da8299ee/specs/), it shows `>` between the path elements instead of a `/`